addon/styles/components/freestyle/usage.scss

```diff
@@ -110,8 +110,9 @@
   }
 
   &-jsonViewer {
```

Comment:
Keep the background and padding the same as when using `ember-json-viewer`. Removed the `font-size` as it seems it wasn't having any effect on the styling.